app/assets/javascripts/admin/onboarding.js

```diff
@@ -147,27 +170,36 @@ const OrganizationForm = Form.create()(({ form, onComplete }) => {
             {getFieldDecorator("organizationName", {
               rules: [{ required: true, message: "Please enter an organization name!" }],
             })(
-              <Input
-                size="large"
+              <AutoComplete
+                dataSource={organizations}
```

Comment:
Is there something, such as `size="large"` for `AutoComplete`? The old input box had the same height and font size as the button next to it. Now, it looks a bit weird :) See https://demo.webknossos.org/onboarding for comparison.